web/sites/default/config/core.base_field_override.node.event.status.yml

```diff
@@ -11,7 +11,7 @@ bundle: event
 label: Published
 description: ''
 required: false
-translatable: false
```

Comment:
Similar to alert status, I think an event status is going to have a translated value, but not a different value.

Reply:
This can also be ignored.